py3status/modules/do_not_disturb.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""
+A simple "Do Not Disturb" module that can turn on and off all system notifications.
+
+A left mouse click will toggle the state of this module.
+
+Configuration parameters:
+    format_off: Display format when the "Do Not Disturb" mode is disabled.
+        (default 'OFF')
+    format_on: Display format when the "Do Not Disturb" mode is enabled.
+        (default 'ON')
+    notification_manager: The process name of your notification manager.
+        (default 'dunst')
+    refresh_interval: Refresh interval to use for killing notification manager process.
+        (default 0.25)
+
+Color options:
+    color_bad: "Do Not Disturb" mode is enabled.
+    color_good: "Do Not Disturb" mode is disabled.
+"""
+
+from time import sleep
+from os import system
+from threading import Thread, Event
+
+
+class Py3status:
+    """
+    """
+
+    # available configuration parameters
+    format_off = 'OFF'
+    format_on = 'ON'
```

Comment:
Up to you, if you like it, I can change :wink: 

I took the current configuration from the example which I saw in my own `.i3status.conf`:

```
wireless wlp3s0 {
  format_up = ""
  format_down = ""
}
```